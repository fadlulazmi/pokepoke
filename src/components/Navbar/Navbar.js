import React from 'react'
import { Link } from 'react-router-dom'
import './styles.scoped.css'

export default function Navbar() {
  const home = '/'
  const myPokemon = '/myPokemon'
  return (
    <nav className="navbar">
      <Link to={home}><p> Home </p></Link>
      <Link to={myPokemon}><p> My Pokemon </p></Link>
    </nav>
  )
}
