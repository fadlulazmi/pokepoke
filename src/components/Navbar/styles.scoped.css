.navbar {
  background-color: lightgreen;
  height: 60px;
  display: flex;
  align-items: center;
  justify-content: center;
}

.navbar > a > p {
  margin: 0;
}