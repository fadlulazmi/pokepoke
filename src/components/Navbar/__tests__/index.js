import Navbar from '../Navbar'
import ShallowRenderer from 'react-test-renderer/shallow';


describe('src/components/Navbar', () => {
  test('should be rendered correctly', () => {
    const shallow = new ShallowRenderer();
    const tree = shallow.render(<Navbar />);
    expect(tree).toMatchSnapshot();
  })
})
