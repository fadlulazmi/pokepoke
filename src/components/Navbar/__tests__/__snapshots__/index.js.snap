// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/Navbar should be rendered correctly 1`] = `
<nav
  className="navbar"
>
  <Link
    to="/"
  >
    <p>
       Home 
    </p>
  </Link>
  <Link
    to="/myPokemon"
  >
    <p>
       My Pokemon 
    </p>
  </Link>
</nav>
`;
